docs/modules/data-connections/pages/data-connections-configuration.adoc

```diff
@@ -381,6 +381,147 @@ OPTIONS (
 --
 ====
 
+[[Hazelcast]]
+=== Example Hazelcast Data Connection
+
+This example configuration shows a data connection to a remote Hazelcast cluster.
+Hazelcast data connection can be used from Pipeline API in link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/pipeline/Sources.html#remoteMapJournal-java.lang.String-com.hazelcast.jet.pipeline.DataConnectionRef-com.hazelcast.jet.pipeline.JournalInitialPosition-com.hazelcast.function.FunctionEx-com.hazelcast.function.PredicateEx-[Sources#remoteMapJournal] source.
+
+NOTE: While Hazelcast data connection can be created using SQL command it cannot yet be used from SQL, because there
+is no SQL connector for remote Hazelcast available.
+
+[tabs]
+====
+XML::
++
+--
+[source,xml]
+----
+<hazelcast>
+  <data-connection name="my-remote-hazelcast">
+    <type>Hz</type>
+    <properties>
+      <property name="client_xml"> <1>
+        <![CDATA[ <2>
+          <?xml version="1.0" encoding="UTF-8"?>
+          <hazelcast-client xmlns="http://www.hazelcast.com/schema/client-config"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
+              http://www.hazelcast.com/schema/client-config/hazelcast-client-config-5.3.xsd">
```

Comment:
@Serdaro @angelasimms 
This should match the minor version of running hazelcalcast cluster, we have `{full-version}` but I didn't find anything like `{minor-version}`